astropy/io/misc/ecsv.py

```diff
@@ -409,7 +410,10 @@ def get_data_kwargs(
 
         kw["na_values"] = null_values
         kw["keep_default_na"] = False
-        kw["skiprows"] = header.n_header
+        kw["comment"] = "#"
+        # Would prefer `kw["skiprows"] = header.n_header` here, but there is a bug in
```

Comment:
please rephrase this comment slightly. As it is, it's not clear what it applies to (the previous line, or the next ? maybe neither ?)